java/src/main/native/src/map_lookup.hpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/column/column.hpp>
+
+namespace cudf {
+
+namespace jni {
+
+  std::unique_ptr<column> map_lookup(
```

Comment:
nit: some kind of documentation for what is supported would be nice, but it is not a big deal

Reply:
Right you are. I just realized that I'd left it out.

Added.